depends/packages/boost.mk

```diff
@@ -1,8 +1,8 @@
 package=boost
-$(package)_version=1_64_0
-$(package)_download_path=https://dl.bintray.com/boostorg/release/1.64.0/source/
+$(package)_version=1_66_0
+$(package)_download_path=https://dl.bintray.com/boostorg/release/1.66.0/source/
 $(package)_file_name=$(package)_$($(package)_version).tar.bz2
-$(package)_sha256_hash=7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332
+$(package)_sha256_hash=5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9
```

Comment:
Is bumping the boost version required?

Reply:
Mmm, Travis is unhappy with the armv7 cross compile. @TheCharlatan any idea? I'm also mystified why Boost 1.64 works fine for our regular aarch64 builds, but why iOs is more picky.